Example Error

When I run `go test` I get the following error:

```
$ go test
# github.com/pkg/sftp_test [github.com/pkg/sftp.test]
./example_test.go:25:8: undefined: sftp.Close
FAIL	github.com/pkg/sftp [build failed]
```

Travis CI is also experiencing an error in testing the master branch.
https://travis-ci.org/github/pkg/sftp/jobs/697375435#L272

--- a/example_test.go
+++ b/example_test.go
@@ -22,7 +22,7 @@ func Example() {
 	if err != nil {
 		log.Fatal(err)
 	}
-	defer sftp.Close()
+	defer client.Close()
 
 	// walk a directory
 	w := client.Walk("/home/user")

